cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -182,7 +185,7 @@ checkLiveViewrequiredTracers traceConfig = do
   if all (== True) reqTracers
   then pure ()
   else do putTextLn "for full functional 'LiveView', please turn on the following \
-                    \tracers in the configuration file: TraceBlockFetchDecisions, \\
+                    \tracers in the configuration file: TraceBlockFetchDecisions, \
```

Comment:
This is the substantive fix.

Reply:
I moved the above function to its own module to avoid `CPP` language extension interference.